test/commands/tooling_job/process_test.rb

```diff
@@ -39,4 +39,82 @@ class ToolingJob::ProcessTest < ActiveSupport::TestCase
 
     ToolingJob::Process.(id)
   end
+
+  test "proxies to analyzer" do
+    id = SecureRandom.uuid
```

Comment:
For now, I've just copy/pasted/modified the existing test runner test. Let me know if you'd like me to use a `setup` method.

Reply:
No, this is fine for now.